TicTacToe.Tests/GameTests.cs

```diff
@@ -7,26 +7,38 @@ namespace TicTacToe.Tests
     public class GameTests
     {
         Game game = new Game();
+        Player player1 = new Player("first", 'X');
+        Player player2 = new Player("second", 'O');
 
         [Fact]
         public void CanCreateNewGame() => Assert.IsType<Game>(game);
 
         [Fact]
-        public void GameHasABoard() => Assert.IsType<String[]>(game.Board);
+        public void GameHasABoard() => Assert.IsType<Char[]>(game.Board);
```

Comment:
When I see a change like this (String to Char), it makes me think that this particular assertion is testing a implementation detail. How can you adjust this test so that it won't need to be updated every time your underlying data structure changes (aka, implementation)?

Reply:
I could simply test that the game has a Board property (regardless of the type of array). The definition of the Board in the Game class is the only place that really needs to know the type, and I'm only testing that a game knows about its board.